pkg/logql/bench/cmd/generate/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/grafana/loki/v3/pkg/logql/bench"
+)
+
+func main() {
+	var (
+		size     = flag.Int64("size", 2147483648, "Size in bytes to generate")
+		dir      = flag.String("dir", "data", "Output directory")
+		tenantID = flag.String("tenant", "test-tenant", "Tenant ID")
+	)
+	flag.Parse()
+
+	// Clean the output directory
```

Comment:
Do we always want to do clear the output dir? I think it is surprising behaviour for a CLI tool.